iceberg/schema.cpp

```diff
@@ -45,4 +46,241 @@ void Schema::FilterColumns(const std::unordered_set<int>& ids_to_remove) {
   fields_.erase(it, fields_.end());
 }
 
+bool IcebergToParquetSchemaValidator::Ensure(bool cond, const std::string& message, std::string& error_log) {
+  if (!cond) {
+    error_log += message;
```

Comment:
Let's append "\n" here, not pass them in message. Newline is a formatting symbol. It's better to split meaning of message and its formatting. So formatting could be changed without changes in function calls.